src/routes/privacy/+page.svelte

```diff
@@ -73,8 +73,22 @@
 
 	<p>
 		E-post sammen med kontaktopplysninger på mottatte og sendte e-poster kan lagret opptil tre år.
-		E-post vil automatisk slettes ved avslutting av kontoer eller etter tre år. Personopplysninger
-		lagres kun i kontrollerte filområder, hvor nedlasting og annen distribusjon er sperret.
```

Comment:
> Personopplysninger lagres kun i kontrollerte filområder, hvor nedlasting og annen distribusjon er sperret.

Jeg syntes denne setningen ble litt vanskelig å forstå, så for å unngå misforståelser så foreslår jeg at vi tar den bort.